client/my-sites/customer-home/free-photo-library-card/style.scss

```diff
@@ -18,9 +18,20 @@
 .free-photo-library-card__demonstration-video {
 	max-height: 600px;
 	max-width: 800px;
-	width: 100%;
-	
-	&:focus {
-		outline: none;
+	width: calc( 90vw - 16px * 2 );
+	position: relative;
+	padding-bottom: calc( 9 / 16 * 100% );
+	height: 0;
+
+	@include breakpoint( '>480px' ) {
+		width: calc( 90vw - 24px * 2 );
```

Comment:
- `90vw`: [Dialog width](https://github.com/Automattic/wp-calypso/blob/146f5ae230fb3e966fd9d53949a55447390b4c8b/packages/components/src/dialog/style.scss#L35).
- `24px`:  [Dialog padding in bigger screens](https://github.com/Automattic/wp-calypso/blob/146f5ae230fb3e966fd9d53949a55447390b4c8b/packages/components/src/dialog/style.scss#L48).